ts/store/reducers/authentication.ts

```diff
@@ -39,13 +42,15 @@ export type LoggedInWithoutSessionInfo = Readonly<{
   kind: "LoggedInWithoutSessionInfo";
   idp: IdentityProvider;
   sessionToken: SessionToken;
+  logoutState: LogoutState;
 }>;
 
 // The user is logged in and we also have all session info
 export type LoggedInWithSessionInfo = Readonly<{
   kind: "LoggedInWithSessionInfo";
   idp: IdentityProvider;
   sessionToken: SessionToken;
+  logoutState: LogoutState;
```

Comment:
Is it needed?
Could we just add another kind? i.e **_LogoutRequested_**
cc @valentinacala

Reply:
mmm yes i added another kind, I think it's the right solution